src/coreclr/inc/switches.h

```diff
@@ -158,3 +158,13 @@
 #endif
 
 #define FEATURE_MINIMETADATA_IN_TRIAGEDUMPS
+
+#ifdef FEATURE_VIRTUAL_STUB_DISPATCH
+#define CHAIN_LOOKUP
+#endif // FEATURE_VIRTUAL_STUB_DISPATCH
+
+#if defined(TARGET_X86)
+// If this is uncommented, leaves a file "StubLog_<pid>.log" with statistics on the behavior
+// of stub-based interface dispatch.
+//#define STUB_LOGGING
+#endif
```

Comment:
```suggestion
// If this is uncommented, leaves a file "StubLog_<pid>.log" with statistics on the behavior
// of stub-based interface dispatch.
//#define STUB_LOGGING
```
I do not think this is x86 specific.

Also, we can delete:
```
src\coreclr\vm\amd64\virtualcallstubcpu.hpp(20)://#define STUB_LOGGING
src\coreclr\vm\arm\virtualcallstubcpu.hpp(13)://#define STUB_LOGGING
```

Reply:
Done